tools/syz-lore/query_lkml.go

```diff
@@ -130,18 +139,10 @@ func processArchives(dir string, emails, domains []string) []*lore.Thread {
 	for i := 0; i < threads; i++ {
 		g.Go(func() error {
 			for rawMsg := range messages {
-				body, err := rawMsg.Extract()
+				msg, err := rawMsg.Parse(emails, domains)
```

Comment:
Please commend why we're not logging the errors here.

Reply:
Added a comment + now it's printing the count of skipped messages (just in case).